### ISSUE TITLE
Use WiFi only doesn't work

### ISSUE THREAD

**Issue by 532910** [BelledonneCommunications/linphone-android#587]
![image](https://user-images.githubusercontent.com/14617699/56475839-c84ef300-6496-11e9-92fe-69624cf51f87.png)

**Comment by rtreleaven** [comment]
failed to reproduce on
Linphone 4.1 (4120)
Linphone SDK 4.1-366-g1b22291
(master)

**Comment by 532910** [comment]
I use exactly the same version.

**Comment by 532910** [comment]
> * call established successfully with bi-directional audio

But it should not be established.

**Comment by 532910** [comment]
With `Use WiFi only` in the debug log I got:
```
[linphone-android] MESSAGE [Platform Helper] Active network type is MOBILE, state CONNECTED / CONNECTED
```

**Comment by rtreleaven** [comment]
I stand corrected.
I have reproduced this bug.
Great bug report btw.

**Comment by Viish** [comment]
Thanks for reporting it.
I fixed it, you need linphone-android commit f6115014d8eece49f76ec4f4bbb0203fe8b10a46 and linphone-sdk commit e4ad24ebd761913db97f587fe457a856a597bf92 (that simply updates linphone library to c86fac300181eef4dc5d963740df7fe786244e7c).

**Comment by Norbert80** [comment]
This is broken again in version 5.2.5. See #2217.
